xonsh/xontribs.py

```diff
@@ -236,16 +324,48 @@ def _list(
         nname = max([6] + [len(x) for x in data])
         s = ""
         for name, d in data.items():
-            lname = len(name)
-            s += "{PURPLE}" + name + "{RESET}  " + " " * (nname - lname)
+            s += "{PURPLE}" + name + "{RESET}  " + " " * (nname - len(name))
             if d["loaded"]:
-                s += "{GREEN}loaded{RESET}"
+                s += "{GREEN}loaded{RESET}" + " " * 4
             else:
                 s += "{RED}not-loaded{RESET}"
+            s += "  "
+            if d["auto"]:
+                s += "{GREEN}auto{RESET}"
+            else:
```

Comment:
```suggestion
            elif d["loaded"]:
```

I think this might be a bit better visually -- so we only display `auto` or `manual` for a given `xontrib` when it's loaded

Reply:
you mean having only two columns and we print auto-loaded and manually-loaded ?

Reply:
No, I mean we should leave the row blank if the xontrib isn't loaded at all, e.g.

```
abbrevs             loaded      manual
autovox             not-loaded  
bashisms            not-loaded  
conda               loaded      manual
coreutils           not-loaded  
direnv              loaded      manual
fish_completer      not-loaded  
free_cwd            not-loaded  
goenv               not-loaded  
langenv_common      not-loaded  
nodenv              not-loaded  
pdb                 not-loaded  
prompt_ret_code     not-loaded  
prompt_starship     not-loaded  
pyenv               not-loaded  
rbenv               not-loaded  
term_integration    not-loaded  
vox                 not-loaded  
voxapi              not-loaded  
whole_word_jumping  not-loaded  
xog                 not-loaded 
```

Reply:
I have moved the code to run when it is loaded. so that it will work when auto-loading is disabled